src/ralph/data_importer/tests.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+from django.test import TestCase
+
+from ralph.data_importer.management.commands import importer
+from ralph.data_importer.resources import AssetModelResource
+
+
+class DatraImporterTestCase(TestCase):
```

Comment:
could you provide simple test with importing some not-basic model from csv (ex. with custom widget, named foreign key etc)?